src/internal/operations/manifests_test.go

```diff
@@ -607,19 +662,22 @@ func (suite *OperationsManifestsUnitSuite) TestProduceManifestsAndMetadata() {
 
 			assert.Len(t, dcs, len(test.expectDCS))
 			for _, dc := range test.expectDCS {
-				var (
-					found bool
-					s     = dc.FullPath().String()
-				)
+				mc, ok := dc.(mockColl)
+				assert.True(t, ok)
+
+				var found bool
 
 				for _, r := range dcs {
-					if r.FullPath().String() == s {
+					rmc, ok := r.(mockColl)
+					assert.True(t, ok)
+
+					if rmc.id == mc.id {
 						found = true
 						break
 					}
 				}
 
-				assert.True(t, found, "expected collection is present in results: "+s)
+				assert.True(t, found, "expected collection is present in results: "+mc.id)
 			}
```

Comment:
not sure if this would be any easier, but you could track the expected IDs of collections as `[]string` and then just iterate through the returned collections, add their IDs to another `[]string` and then just call `assert.ElementsMatch(t, expectedIDs, foundIDs)`

Reply:
good thinking